modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc

```diff
@@ -30,7 +30,7 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_cylfrac.ico"
 #ifdef LANGUAGE_ES_ES
     #include "lang/es-ES.rc"
 #endif
-#ifdef LANGUAGE_ES_ES
+#ifdef LANGUAGE_FR_FR
```

Comment:
Thanks ;)

Reply:
Even if I'm not a hacker when I read it I realized there was a problem :)